server/container_stop.go

```diff
@@ -82,5 +80,15 @@ func (s *Server) stopContainer(ctx context.Context, ctr *oci.Container, timeout
 		log.Warnf(ctx, "Unable to write containers %s state to disk: %v", ctr.ID(), err)
 	}
 
+	if hooks != nil {
+		if err := hooks.PostStop(ctx, ctr, sb); err != nil {
+			return fmt.Errorf("failed to run post-stop hook for container %q: %w", ctr.ID(), err)
```

Comment:
This must be just a log. The container is dead and there is nothing that can be done. Otherwise it will confuse the platform. See https://github.com/cri-o/cri-o/pull/7032

Reply:
In fact.. I wonder if it should have been below the stopContainer...

Reply:
I cherry-picked https://github.com/cri-o/cri-o/pull/7013/commits/93c4c561298e5e6fbc1a8dcaf9cfc3be7aab5e89